src/main/java/org/intellij/sdk/codesync/CodeSyncLogger.java

```diff
@@ -93,11 +94,11 @@ private static void logMessageToCloudWatch(
         }
     }
 
-    public static void logEvent(String message) {
-        logEvent(message, null);
+    public static void logEvent(String message, String type) {
+        logEvent(message, null, type);
```

Comment:
@saleem-latif Question, Since we are using userEmail as Log Stream name, to which log stream it sends the logs when userEmail is null?